microraiden/microraiden/examples/echo_client.py

```diff
@@ -39,7 +37,7 @@ def run(
         key_path: str,
```

Comment:
Can we change `--key-path` to `--private-key`? Reason: consistency with the other examples.